packages/xdl/src/detach/IosNSBundle.ts

```diff
@@ -32,6 +32,22 @@ const DEFAULT_FABRIC_KEY = '81130e95ea13cd7ed9a4f455e96214902c721c99';
 const DEFAULT_GAD_APPLICATION_ID = 'ca-app-pub-3940256099942544~1458002511';
 const KERNEL_URL = 'https://expo.io/@exponent/home';
 
+export function getManifestFileNameForSdkVersion(sdkVersion: string) {
```

Comment:
Why are these methods defined twice?

Reply:
Once per platform, but I am happy to extract them to a separate util module